src/stream-ciphers.rst

```diff
@@ -137,43 +137,42 @@ significantly fewer pixels per block than the larger block sizes in the
 example. But AES is the workhorse of modern block ciphers—it can't be at
 fault, certainly not because of an insufficient block size.
 
-When we look at a picture of what would happen with an idealized
-encryption scheme, we notice that it looks like random noise. Keep in
-mind that “looking like random noise” doesn't mean something is properly
-encrypted: it just means that we can't inspect it using methods this
-trivial.
+Notice that an idealized
+encryption scheme looks like random noise.
+“Looking like random noise” does not mean something is properly
+encrypted: it just means that we cannot inspect it using trivial methods.
```

Comment:
For all these paragraphs: please consider M-q / filling these paragraphs to their previous consistent line length :)

Reply:
I ran M-q on above paragraph (line 140 - 143)

When you said "these paragraphs", do you mean M-q on all other paragraphs as well?

Reply:
Nah just the ones changed in this PR. Thanks!